testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergProcedureCalls.java

```diff
@@ -59,6 +60,103 @@ public void testMigrateHiveTable()
         onTrino().executeQuery("DROP TABLE IF EXISTS " + icebergTableName);
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testMigrateTimestampHiveTableWithOrc()
+    {
+        asList(
```

Comment:
The helper method contains several assertions. We won't be able to find the failed test case easily with `forEach`. I prefer calling one-by-one in this case. 
```java
        testMigrateTimestampHiveTable(
                "MILLISECONDS",
                3,
                "2021-01-01 10:11:12.123",
                "2021-01-01 10:11:12.123000",
                "2021-01-01 10:11:12.123",
                "ORC");
        testMigrateTimestampHiveTable(
                "MICROSECONDS",
                6,
                "2021-01-01 10:11:12.123456",
                "2021-01-01 10:11:12.123456",
                "2021-01-01 10:11:12.123456",
                "ORC");
...
```